### PR TITLE
Implement new main menu and updated header

### LINKED ISSUE #7010: T - Implement new main menu for mobile
In https://github.com/mdn/kuma/issues/6784 a new hamburger style menu have been designed for mobile devices.

79573164-315a4e00-808c-11ea-82b7-86dfc6bab7c6

## Interaction on mobile

1. Menu is initially collapsed behind a hamburger style trigger icon
2. When tapped, the icon changes slightly and the menu is shown with the signin/avatar menu entry as well as the other top-level main menu entries
3. Tapping on any of these will expand the relevant submenu
4. Tapping again will collapse the submenu.
5. Tapping a different top-level menu item will expand its submenu and collapse all other
6. Tapping the 🍔 again, will collapse the entire menu and switch the icon back to the standard 🍔 menu icon.

For tablet and desktop the main aspects of the menu remain the same with some subtle design updates. See https://github.com/mdn/kuma/issues/6784 for details.

## Acceptance Criteria

- [ ] Mobile version of menu is implemented as described above
- [ ] Relevant design changes made to main menu on tablet and desktop
- [ ] Ensure menu is keyboard accessible
- [ ] instrumentation matches desktop events or implements new events

### PR DESCRIPTION
**Pull request by schalkneethling**
Implements new main menu. Also cleaned up our page header and some other small tweaks.

@mindy r?

fix #7010

### CODE REVIEW

**Comment by schalkneethling** [comment]
Whoops! Found a small issue. Resolving.

**Comment by schalkneethling** [comment]
Ready for r?

File: kuma/static/styles/minimalist/components/_main-nav.scss [hunk 1]
```diff
@@ -4,19 +4,32 @@
...
+    position: absolute;
+    top: 80px;
+    left: 0;
+    background-color: #fff;
+    padding: 20px;
+    width: 100%;
+    height: 100vh;
+    font-weight: bold;
+    z-index: $menu-zindex;
+
```
Comment (mindy) [inline kuma/static/styles/minimalist/components/_main-nav.scss:21]: Can you add a `box-sizing: border-box` so we don't have a horizontal scroll caused by the padding and 100% width?

File: kuma/static/styles/minimalist/components/_main-nav.scss [hunk 2]
```diff
@@ -96,4 +109,8 @@
             }
         }
     }
+
+    &.show {
```
Comment (mindy) [inline kuma/static/styles/minimalist/components/_main-nav.scss:113]: I think you need to add something to override the `display: block` on tablet and up. If I open the menu and resize, it looks like this:
Screen Shot 2020-05-20 at 3 13 04 PM

Comment (schalkneethling) [reply kuma/static/styles/minimalist/components/_main-nav.scss:113]: Hmmm, strange. I have not seen this. Will take a look.

Comment (schalkneethling) [reply kuma/static/styles/minimalist/components/_main-nav.scss:113]: Now I understand what you mean! Nice catch @mindy :tea:  Fixed.

**Comment by schalkneethling** [comment]
Thank you for the review @mindy - Updated.

Also, tested on iOS and Android:

## iOS





## Android

**Pull request merged**